eureka2-write-server/src/main/java/com/netflix/eureka2/server/EurekaWriteServerModule.java

```diff
@@ -104,5 +102,7 @@ public void configureEureka() {
         bind(SelfRegistrationService.class).to(EurekaWriteServerSelfRegistrationService.class).asEagerSingleton();
 
         bind(AbstractEurekaServer.class).to(EurekaWriteServer.class);
+
+        install(new OverridesModule());
```

Comment:
This module is installed directly in WriteServerRunner. It is not installed in the embedded cluster.